docs/source/en/quicktour.mdx

```diff
@@ -389,3 +389,42 @@ One particularly cool 🤗 Transformers feature is the ability to save a model a
 ```
 </tf>
 </frameworkcontent>
+
+## Custom model builds
+
+You can modify the model's configuration class to change how a model is built. The configuration specifies a model's attributes, such as the number of hidden layers or attention heads. You start from scratch when you initialize a model from a custom configuration class. The model attributes are randomly initialized, and you'll need to train the model before you can use it to get meaningful results.
+
+Start by importing a model's configuration class, and then you can change the number of attention heads, for instance:
+
+```py
+>>> from transformers import DistilBertConfig
+
+>>> my_config = DistilBertConfig(n_heads=12)
```

Comment:
We haven't used any specific model class up until know, only the auto classes. Maybe it's best to use auto classes here too?